arch/arm/boot/dts/qcom/batterydata-yukon-flamingo.dtsi

```diff
@@ -72,7 +72,7 @@ qcom,flamingo-batterydata {
 	};
 
 	qcom,rbatt-sf-lut {
-		qcom,lut-col-legend = <(-20) 0 25 40 60 70>;
```

Comment:
I'm pretty sure this value was added by mistake. I checked in kernel 18.5.C.0.25 and it is not there.
Besides that, if the value is correct, then there would be some data missing in qcom,lut-data.